### PR TITLE
Include all autoconfigure exlucdes when running in local dev mode.

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

**Comment by zmoore-usgs** [comment]
Going to go ahead and merge since it succeed on Travis and this isn't really a functional change.

**Comment by mhines-usgs** [comment]
pulling this locally to try again !

**Comment by mhines-usgs** [comment]
Worked great!

File: Dockerfile [hunk 1]
```diff
@@ -51,6 +51,7 @@ ENV localOauthClientGrantTypes="authorization_code, access_token, refresh_token,
 ENV localOauthClientScopes=user_details
 ENV localOauthResourceId=local-app
 ENV localUserName=user
+ENV localUserPassword=chanegMe
```
Comment (mhines-usgs) [inline Dockerfile]: does this typo matter?

**Pull request merged**